src/app/js/admin/upload/index.js

```diff
@@ -18,9 +18,11 @@ export const closeUpload = createAction(CLOSE_UPLOAD);
 export const changeUploadUrl = createAction(CHANGE_UPLOAD_URL);
 export const changeLoaderName = createAction(CHANGE_LOADER_NAME);
 
-const validateUrl = url =>
-    console.log({ url }) ||
-    (url && (url.startsWith('http://') || url.startsWith('https://')));
+const validateUrl = url => {
+    if (url && (url.startsWith('http://') || url.startsWith('https://')))
```

Comment:
Just return the condition will works, "if" is useless here

Reply:
There was a bug sayingthe return type is not always boolean with this syntax